New Term: Java `Math.random()` (#884)
Codecademy/docs: content/java/concepts/math-methods/terms/atan/atan.md

  - 'paths/computer-science'
---

> The **`Math.atan()`** method returns the inverse tangent of the argument in radians.

Reviewer: ```suggestion
The **`Math.atan()`** method returns the inverse tangent (or arctangent) of the argument in radians.
```

## Syntax

```pseudo
Math.atan(n)
```

> 

Reviewer: Can we add a brief sentence here describing what `n` is (including what data type(s))?

Reviewer: Also, on line # 24, can we rephrase the sentence to something like this?

```
The following example uses `Math.atan()` to return the inverse tangent of `1.0`:
```

  - 'paths/computer-science'
---

The **`Math.atan()`** method returns the inverse tangent (or arctangent) of the argument in radians.

## Syntax

```pseudo
Math.atan(n)
```

> `n` is an `double` whose arctangent is to be returned.

Reviewer: ```suggestion
`n` is a `double` whose arctangent is returned.
```
